### DISCUSSION TITLE
VMs works on same cloudlets

### DISCUSSION THREAD

**Discussion by HilmyThoriq** [manoelcampos/cloudsim-plus#450] [Q&A] [answered]
Hello,

I want to ask is it possible in cloudsimplus that vm works on the same cloudlet ? let say i have 2 VM and want to work the same cloudlet at the same time.

**Comment by manoelcampos** [comment, marked as answer]
Think of cloudlets as processeses. This way, you cannot have the same cloudlet instance inside different VMs.
But two different processesses can be started from the same application. If you want to simulate that, you just need to create different cloudlet instances with the same configuration. Each Cloudlet must have a different ID (such as different processesses have different PIDs).
